pkg/tsdb/sqleng/sql_engine.go

```diff
@@ -283,6 +283,10 @@ func (e *dataPlugin) executeQuery(query plugins.DataSubQuery, wg *sync.WaitGroup
 			errAppendDebug("db has no time column", errors.New("no time column found"), interpolatedQuery)
 			return
 		}
+
+		// Make sure to name the time field 'Time' to be backward compatible with Grafana pre-v8.
+		frame.Fields[qm.timeIndex].Name = "Time"
```

Comment:
```suggestion
		frame.Fields[qm.timeIndex].Name = data.TimeSeriesTimeFieldName
```
maybe use the constant so it is more clear that the name is meaningful?

Reply:
Thanks :+1: